src/components/Hero.tsx

```diff
@@ -1,21 +1,52 @@
 'use client';
 
-import { useContext } from 'react';
+import { useContext, useEffect, useState } from 'react';
 import { useRouter } from 'next/navigation';
+import Image from 'next/image';
+import { isSameMonth } from 'date-fns';
 import { AuthContext, IAppContext } from '@/app/context/auth.context';
 import { APP } from '@/utils/app.constants';
 import Button from '@/components/Button';
+import { IUserDataContext, UserDataContext } from '@/app/context/userData.context';
+import ModalCreateNewMonth from './ModalCreateNewMonth';
 
 const Hero = () => {
+  const [isCurrentMonthOpen, setIsCurrentMonthOpen] = useState<boolean>(false);
+  const [isModalOpen, setIsModalOpen] = useState<boolean>(false);
+  const { userMonths } = useContext(UserDataContext) as IUserDataContext;
   const { user } = useContext(AuthContext) as IAppContext;
   const router = useRouter();
 
+  let userIsLoggedInAndCurrentMonthIsNotOpen = user && !isCurrentMonthOpen;
+
+  //check if current month is open
+  useEffect(() => {
+    if (userMonths.length > 0) {
+      let month = userMonths.find((oneMonth, i) => {
+        if (isSameMonth(new Date(), oneMonth.createdAt)) {
+          return oneMonth;
+        }
+      });
+      if (month) {
+        setIsCurrentMonthOpen(true);
+      }
+    }
+  }, [userMonths]);
+
   return (
     <section className='w-full md:w-80'>
-      {!user && (
+      {(!user || (user && userMonths.length < 1)) && (
         <>
-          <div className='w-full md:w-80 h-80 bg-slate-400 mb-4'></div>
-          <Button clickHandler={() => router.push(APP.pageRoutes.login)}>{APP.buttonAction.seeMonths}</Button>
+          {(!user || userIsLoggedInAndCurrentMonthIsNotOpen) && (
+            <div className='w-full md:w-80 h-80 mb-4'>
+              <Image src='/images/budget-transparent-medium.png' width={320} height={320} alt='Colorful machine managing the money' priority />
```

Comment:
Maybe you could put the img path in app.constants file or create a component if you are going to reuse the same img.